### DISCUSSION TITLE
Welcome to berry Discussions!

### DISCUSSION THREAD

**Discussion by skiars** [berry-lang/berry#185] [Announcements]
## 👋 Welcome!
  We're using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you're wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by s-hadinger** [comment]
Nice to see you're back!!! The Tasmota community already loves Berry, and it's become an excellent alternative to MicroPython (in my unbiased opinion...)

**Comment by morrowwm** [comment]
Anywhere I can go to read about the beginning of Berry? Why was it created? Who was involved? Just curious.

**Reply by s-hadinger** [reply]
See the preface: https://github.com/berry-lang/berry/wiki/Reference

It was created by Guan Wenliang, and ported to Tasmota by myself. I have been contributing to Berry and fixing some bugs since then.

**Reply by morrowwm** [reply]
Thank you. Interesting story!

**Comment by Yechk** [comment]
Great project you have here. I'm writing the build system for my c++20 project in Berry right now. I love how lightweight/easy to embed it is.
